test/test_posv.cc

```diff
@@ -266,10 +290,29 @@ void test_posv_work(Params& params, bool run)
         params.time() = time;
         params.gflops() = gflop / time;
 
-        if (timer_level >= 2) {
+        if (timer_level >= 2 && params.routine == "posv") {
             params.time2() = slate::timers[ "posv::potrf" ];
             params.time3() = slate::timers[ "posv::potrs" ];
         }
+        else if (timer_level >=2 && params.routine == "posv_mixed") {
+            params.time2() = slate::timers[ "posv_mixed::potrf_lo" ];
+            params.time3() = slate::timers[ "posv_mixed::potrs_lo" ];
+            params.time4() = slate::timers[ "posv_mixed::hemm_lo" ];
+            params.time5() = slate::timers[ "posv_mixed::add_lo" ];
+            params.time6() = slate::timers[ "posv_mixed::potrf_hi" ];
+            params.time7() = slate::timers[ "posv_mixed::potrs_hi" ];
+        }
+        else if (timer_level >=2 && params.routine == "posv_mixed_gmres") {
+            params.time2() = slate::timers[ "posv_mixed_gmres::potrf_lo" ];
+            params.time3() = slate::timers[ "posv_mixed_gmres::potrs_lo" ];
+            params.time4() = slate::timers[ "posv_mixed_gmres::hemm_lo" ];
+            params.time5() = slate::timers[ "posv_mixed_gmres::add_lo" ];
+            params.time6() = slate::timers[ "posv_mixed_gmres::potrf_hi" ];
+            params.time7() = slate::timers[ "posv_mixed_gmres::potrs_hi" ];
+            params.time8() = slate::timers[ "posv_mixed_gmres::rotations" ];
+            params.time9() = slate::timers[ "posv_mixed_gmres::trsm_lo" ];
+            params.time10() = slate::timers[ "posv_mixed_gmres::gemm_lo" ];
+        }
```

Comment:
Match the `hi` changes.

If timers aren't showing up as expected (i.e., are 0.0), you might try using `timers.at[ "..." ]` instead of `timers[ "..." ]`. The C++ std convention is `at( )` throws key errors, `[ ]` does not.

Reply:
I will double check this, but I didn't see zero's the last time that I ran it. I will post output after all changes have been pushed and finalized. Such that we can decide if possibly more timers needed to be added.

Reply:
Looks good.

The matrix generator doesn't currently support the matrix type we need. What we need is `--matrix poev --cond 1e10`, which can be solved in double but not single.